test/test_autograd.py

```diff
@@ -2666,9 +2666,9 @@ def test_profiler(self):
         self.assertFalse(torch.autograd._profiler_enabled())
 
         last_end = 0
-        names = ['is_complex', 'mul', 'to', 'empty_strided', 'copy_', 'is_complex',
+        names = ['is_complex', 'mul', 'to', 'empty_strided', 'is_complex',
```

Comment:
Why are the copies gone from here? We actually would like to keep these.
Do we need to update the TracingTypeManual implementation to make sure copy_ is still properly tracked?